app/src/main/java/nyc/c4q/jonathancolon/inContaq/contactlist/fragments/MonthlyGraphFragment.java

```diff
@@ -0,0 +1,59 @@
+package nyc.c4q.jonathancolon.inContaq.contactlist.fragments;
+
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.db.chart.view.LineChartView;
+
+import org.parceler.Parcels;
+
+import java.util.ArrayList;
+
+import nyc.c4q.jonathancolon.inContaq.R;
+import nyc.c4q.jonathancolon.inContaq.contactlist.Contact;
+import nyc.c4q.jonathancolon.inContaq.contactlist.activities.ContactListActivity;
+import nyc.c4q.jonathancolon.inContaq.utlities.graphs.MonthlyGraph;
+import nyc.c4q.jonathancolon.inContaq.utlities.sms.Sms;
+import nyc.c4q.jonathancolon.inContaq.utlities.sms.SmsHelper;
+
+
+public class MonthlyGraphFragment extends Fragment {
```

Comment:
DailyGraphFragment and MonthlyGraphFragment should subclass a general GraphFragment class to avoid duplicate code